content/en/apps/features/reports/_index.md

```diff
@@ -37,15 +37,15 @@ The toolbar at the top of the page includes filters and search to help users nar
 - **Date of Submission**
 - **Status** (e.g. not reviewed, has errors, correct, valid SMS, invalid SMS)
 
-Using the search box, you can search for reports by patient name, phone number, ID number and more. To reset the filters or the search and view  the full list of forms, click on the reset icon on the right side of the toolbar.
+Using the search box, you can search for reports by patient name, phone number, ID number and more. To reset the filters and view the full list of forms, click on "Reset" located in the filter sidebar.
 
-{{< figure src="reports-search-reset.png" link="reports-search-reset.png" class="left col-3 col-lg-3" >}}
+{{< figure src="reports-search.png" link="reports-search.png" class="left col-3 col-lg-3" >}}
 
-{{< figure src="reports-search.png" link="reports-search.png" class="left col-9 col-lg-9" >}}
+{{< figure src="reports-search-reset.png" link="reports-search-reset.png" class="left col-9 col-lg-9" >}}
 
 <br clear="all">
 
-{{% alert title="Note" %}} A new user experience for Filter and Search was introduced in v3.17. The previous version can be re-enabled for users by adding the [permission]({{< ref "apps/reference/app-settings/user-permissions" >}}) `can_view_old_reports_filter` to the user's role; however, the old version should be considered deprecated and will be completely removed in a future release. {{% /alert %}}
+{{% alert title="Note" %}} A new user experience for Filter and Search was introduced in v3.17. The previous version can be re-enabled for users by adding the [permission]({{< ref "apps/reference/app-settings/user-permissions" >}}) `can_view_old_filter_and_search` to the user's role; however, the old version should be considered deprecated and will be completely removed in a future release. {{% /alert %}}
```

Comment:
How about this?

....however, the old version should be considered deprecated and will be completely removed in a future release. See [Feature Flags](https://docs.communityhealthtoolkit.org/apps/guides/updates/feature-flags/) documentation for more info.